src/device-registry/utils/scripts/api-response-transformer.js

```
@@ -7,6 +7,21 @@
 const axios = require("axios");
 const isEmpty = require("is-empty");
 
+function filterWithoutSiteCategoryAndInvalidCoordinates(sites) {
+  const result = sites
+    .filter(
+      (site) =>
+        !site.hasOwnProperty("site_category") &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)